settings/Screen_Height_200.txt

```diff
@@ -397,8 +401,10 @@ SimonTheSorcererAGA
 Simulcra
 SirFredTheLegend
 SoldierOfLight
+Speedball2
```

Comment:
Speedball2 is in the 240 file. Are you sure this fits in 200px ? I seem to recall either the health bar wasnt visible, or one of the menu/title screens ended up cropped?

Reply:
200 looks fine and fills the screen. The status bar is at the bottom. At 240 it leaves a big black bit at the bottom. I imagine a lot of these entries are wrong.

Reply:
i wouldnt be so quick to assume that - there could be an impact from different displays, or changes from amiberry (even release/dev versions) or uae4arm etc.

Reply:
If there are changes between versions of Amiberry and other peoples displays then this will be a neverending process !

Reply:
Yes i agree - which is why this program will take advantage of auto-scaling when possible. 

On this basis, where things are in dispute, or very close, it is better to "take the hit" of having a screen one step bigger.